app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/PublisherTakeOverBundle.kt

```diff
@@ -58,6 +62,27 @@ import com.aptoide.android.aptoidegames.theme.Palette
 import kotlin.random.Random
 import kotlin.random.nextInt
 
+internal var toSendImpression = false
```

Comment:
could this flag be inverted ? 
like hasSentImpression maybe it would be easier to manage